client/homepage/stats-overview/style.scss

```diff
@@ -85,7 +85,7 @@ article.woocommerce-stats-overview__install-jetpack-promo {
 			}
 		}
 
-		.woocommerce-layout & button.components-button.is-button.is-primary {
+		.woocommerce-layout & button.components-button.is-primary {
 			margin-left: 24px;
```

Comment:
While we're here we can update this to use `$gap-large` and I noticed a few other places in here that could benefit from using some of these vars.

```scss
$gap-largest: 40px;
$gap-larger: 36px;
$gap-large: 24px;
$gap: 16px;
$gap-small: 12px;
$gap-smaller: 8px;
$gap-smallest: 4px;
```